CDS_Dashboard/script.js

```diff
@@ -314,18 +294,39 @@ function clearModal() {
     });
 }
 
-var counter = 0;
+function buildOptions(drilldown, options) {
+    if (drilldown) { // drill down
+        options = {
```

Comment:
Since we're redefining `options` altogether, you'll need to include the `var` keyword to scope it properly.